test/edu/ucsb/nceas/metacat/restservice/v2/MNResourceHandlerTest.java

```diff
@@ -40,9 +41,12 @@ public class MNResourceHandlerTest {
     /**HTTP Verb HEAD*/
     protected static final byte HEAD = 5;
```

Comment:
HEAD is never used

Reply:
Also - can't these all be `private` instead of `protected`?

Reply:
(we're having some challenges with comments not showing up, so I'm pasting Jing's reply here, to keep the conversation in one place)

>
> HEAD is never used
>

(Jing said) Weird. I added a comment before. This one can be used in the future. I think we have the code to use HEAD.

Reply:
In that case, I think we should add it in future, if and when it is needed, instead of keeping it here now. 

Unused code is (a) a maintenance burden, and (b) confusing to someone else who is trying to understand the tests.

Reply:
I generally agree to your statement regarding unused code. However, this constant will be used for sure in the future. And it is a constant, not adding complexity to the code logic. When we need it in the future, we don't dig around to figure out. So we can save lots of time. What do you think?

Those constants will be used in another test classes, such as CNResourceHandlerTest. So I declare them `protected`.

Reply:
TL;DR - that's fine - leave it in. My opinion has not changed, but I don't think it's worth arguing about any longer 😁